compile error

cargo build, from ev2400... directory within emerald-master.  compiles finally but with error:
   Compiling ti-i2c v0.1.0 (C:\Users\billj\vis_pro\embassy-main\ev2400-on-raspberry-pi-pico-master)
error: git describe exited with status 128: fatal: not a git repository (or any of the parent directories): .git
  --> src/main.rs:67:9
   |
67 |         git_version!(args = ["--tags", "--dirty"])
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `git_version` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0554]: `#![feature]` may not be used on the stable release channel
 --> src/main.rs:3:12
  |
3 | #![feature(future_join)]
  |            ^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `ti-i2c` (bin "ti-i2c") due to 2 previous errors

--- a/src/main.rs
+++ b/src/main.rs
@@ -1,6 +1,5 @@
 #![no_std]
 #![no_main]
-#![feature(future_join)]
 
 mod commands;
 mod indications;
@@ -64,7 +63,7 @@ async fn main(spawner: Spawner) {
 
     info!(
         "ti-i2c ({}) is running. Hello!",
-        git_version!(args = ["--tags", "--dirty"])
+        git_version!(args = ["--tags", "--dirty"], fallback = "unknown")
     );
 
     static LED_INDICATIONS: LedIndications = LedIndications::new();

